.github/scripts/get-reviewers.mjs

```diff
@@ -0,0 +1,67 @@
+import fs from 'fs';
+
+export default async function (core, filesList) {
+  // Load reviewers mapping
+  const reviewersMap = JSON.parse(
+    fs.readFileSync('.github/reviewers.json', 'utf8')
+  );
+
+  // Parse the file list from the input
+  const files = filesList.trim().split(' ').filter(Boolean);
+
+  // Track reviewers we need to mention
+  const reviewersToMention = new Set();
+
+  // Process each file
+  for (const file of files) {
+    const content = fs.readFileSync(file, 'utf8');
+    const apiValues = extractApisFromFrontmatter(content);
+
+    if (apiValues && apiValues.length > 0) {
+      for (const apiValue of apiValues) {
+        if (reviewersMap[apiValue]) {
+          reviewersToMention.add(`- [ ] ${reviewersMap[apiValue]}`);
```

Comment:
Ik this is how core does it, but wouldn't it be better to use the GitHub API to request a review from that team instead of just mentioning them?

Reply:
iirc, a team needs explicit write access in the repo to be able to request a review from them natively

Reply:
According to the docs,

> Organization members with write access can also assign a pull request review to any person or team with read access to a repository.

Yet, if I attempt to request a review from a team w/o access, it doesn’t let me 🤔